src/main/java/org/jboss/aerogear/keycloak/metrics/ResourceExtractor.java

```diff
@@ -54,4 +54,22 @@ static String getResource(UriInfo uriInfo) {
         return "";
     }
 
+    /**
+     * This method obtains a list of resource info from the {@link UriInfo} object and returns the resource URI.
+     * @param uriInfo {@link UriInfo} object obtained from JAX-RS
+     * @return The resource uri.
+     */
+    static String getURI(UriInfo uriInfo) {
+        if (URI_METRICS_ENABLED) {
```

Comment:
does it make sense here to replace certain parts of the URI with a place holder to reduce uniqueness? For example replace ``` /realm/master ``` with ```/realm/{realm}``` for example. Looking for ways to reduce uniqueness while still providing the needed insight

Reply:
That's possible, We could do another flag to enable more detailed realm output if needed ?

Reply:
@CathalOConnorRH if we add that, we can also add the change were we remove the labels completely if URI_METRICS_ENABLED is not set (at the moment they are put on all metrics with an empty value).

Reply:
agreed, I'll work on both of those changes today